Remotion/Web/Development.WebTesting.WebFormsControlObjects/TreeViewNodeControlObject.cs

```diff
@@ -117,7 +118,14 @@ public TreeViewNodeControlObject WithItemID (string itemID)
       private TreeViewNodeControlObject FindAndCreateNodeInHierarchy (string xpathSuffix)
       {
         var nodeScope = GetChildrenScope (_treeViewNode.Scope).FindXPath ("//table" + xpathSuffix);
-        return new TreeViewNodeControlObject (_treeViewNode.Context.CloneForControl (nodeScope));
+        try
+        {
+          return new TreeViewNodeControlObject (_treeViewNode.Context.CloneForControl (nodeScope));
+        }
+        catch (StaleElementException)
+        {
+          throw new WebTestException ("This element has been removed from the DOM.");

```

Comment:
Pass the original exception? If we want to do that we should do it for all thrown exceptions in this PR

Reply:
What would be the benefit?

Reply:
Perhaps it could be helpful to the tester? I formulated it as a question because I'm not sure either

Reply:
I don't think so, but I'll leave that open for the lead review.

Reply:
don't pass GO, go directly to jail.

Reply:
message: do we already have an 'element not found' exception? would this be correct here?

Reply:
@MichaelKetting  Could use `AssertionExceptionUtility.CreateControlMissingException (ex.Message)`. Would be fine to use imo.

Reply:
sounds good.